src/components/ApplicationForm/Skills.js

```diff
@@ -275,6 +276,24 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
             customRef={refs['longAnswers2Ref']}
           />
         </FormGroup>
+        <FormGroup>
+          <QuestionHeading>question 21</QuestionHeading>
+          <SubHeading size="1.25em">Draw your favourite character!</SubHeading>
+          <SketchCanvas
+            width={600}
```

Comment:
I don't think this is mobile responsive right now. Looking at the npm package, it looks like react-sketch-canvas supports web & mobile

Reply:
done (i think). I didn't touch height cuz I personally think it looks better with the same height across all devices, but lmk if you think otherwise!